app/views/layouts/_topbar.html.erb

```diff
@@ -1,27 +1,27 @@
 <% if user_signed_in? %>
```

Comment:
We should check whether we need to show topbar or not before calling to render it.